internal/proxy/task_insert_streaming.go

```diff
@@ -97,19 +97,20 @@ func repackInsertDataForStreamingService(
 			return nil, err
 		}
 		for _, msg := range msgs {
+			insertRequest := msg.(*msgstream.InsertMsg).InsertRequest
 			newMsg, err := message.NewInsertMessageBuilderV1().
 				WithVChannel(channel).
 				WithHeader(&message.InsertMessageHeader{
 					CollectionId: insertMsg.CollectionID,
 					Partitions: []*message.PartitionSegmentAssignment{
 						{
 							PartitionId: partitionID,
-							Rows:        uint64(len(rowOffsets)),
+							Rows:        insertRequest.GetNumRows(),
```

Comment:
Is this right? rowOffsets is hashed, so it should be smaller than insertMsg.NumRows, right?